### PR TITLE
Fixed the possibility of posting multiple signed membership forms.

### PR DESCRIPTION
**Pull request by Jessseee**
Also updated the membership part of the user admin interface.

### CODE REVIEW

File: app/Http/Controllers/UserAdminController.php [hunk 1]
```diff
@@ -54,7 +54,9 @@ public function index(Request $request)
     public function details($id)
     {
         $user = User::findOrFail($id);
-        return view('users.admin.details', ['user' => $user]);
+        $memberships['pending'] = Member::withTrashed()->where('user_id', '=', $user->id)->where('deleted_at', '=', null)->where('pending', '=', true)->get();
+        $memberships['previous'] = Member::withTrashed()->where('user_id', '=', $user->id)->where('deleted_at', '!=', null)->get();
+        return view('users.admin.details', ['user' => $user, 'memberships' => $memberships]);
```
Comment (JonaMata) [inline app/Http/Controllers/UserAdminController.php:59]: It shouldn't happen, but what if a user has multiple non pending non deleted memberships, then only one is shown.

**Review by JonaMata — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/UserAdminController.php (hunk 1 above)
Comment (Jessseee) [reply app/Http/Controllers/UserAdminController.php:59]: Should I just add another line for "broken" memberships? 😂

File: routes/web.php [hunk 2]
```diff
@@ -188,12 +188,15 @@
...
-        Route::post('sign', ['as' => 'sign', 'middleware' => ['auth'], 'uses' => 'UserDashboardController@postMemberForm']);
-        Route::get('{id}', ['as' => 'download', 'uses' => 'UserAdminController@showMemberForm']);
-        Route::post('print/{id}', ['as' => 'print', 'middleware' => ['auth', 'permission:board'], 'uses' => 'UserAdminController@printMemberForm']);
-        Route::post('delete/{id}', ['as' => 'delete', 'middleware' => ['auth', 'permission:board'], 'uses' => 'UserAdminController@destroyMemberForm']);
+    Route::group(['prefix' => 'memberform', 'as' => 'memberform::', 'middleware' => ['auth']], function() {
+        Route::get('sign', ['as' => 'sign', 'uses' => 'UserDashboardController@getMemberForm']);
+        Route::post('sign', ['as' => 'sign', 'uses' => 'UserDashboardController@postMemberForm']);
+        Route::group(['prefix' => 'download', 'as' => 'download::'], function() {
+            Route::get('{id}', ['as' => 'new', 'uses' => 'UserAdminController@getNewMemberForm']);
+            Route::get('{id}', ['as' => 'signed', 'uses' => 'UserAdminController@getSignedMemberForm']);
```
Comment (JonaMata) [inline routes/web.php:196]: These URLs are identical, these should be changed

**Review by JonaMata — CHANGES_REQUESTED** [review]
LAGTM

**Review by JonaMata — APPROVED** [review]
LGTM

**Pull request merged**